enter-metannotate

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```

Comment:
Should be ```#!/usr/bin/env bash```. https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash